src/parser/core/modules/ActionWindow/evaluators/NotesEvaluator.ts

```diff
@@ -6,12 +6,13 @@ import {EvaluationOutput, WindowEvaluator} from './WindowEvaluator'
 
 export abstract class NotesEvaluator implements WindowEvaluator {
 
-	protected abstract header : RotationTarget
+	protected abstract header : RotationTarget | undefined
```

Comment:
This was an artifact of how I originally wrote the RulePassedEvaluator on #1832 since not all of the 'rules' there have column output. It might go away with the re-implementation of RulesPassed as an abstract based on your other comment. will see.

Reply:
still need it for DNC's implementation after all (see current state of #1832)